inspirehep/modules/disambiguation/core/ml/models.py

```diff
@@ -76,3 +102,161 @@ def fit(self):
 
     def predict(self, X):
         return self.estimator.predict(X)
+
+
+class DistanceEstimator(object):
+    def __init__(self, ethnicity_estimator_filename):
+        self.ethnicity_estimator = EthnicityEstimator()
+        self.ethnicity_estimator.load_model(ethnicity_estimator_filename)
+
+    def load_data(self, signatures_filename, records_filename, pairs_filename):
+        signatures, records = load_signatures(signatures_filename, records_filename)
+        pairs = json.load(open(pairs_filename, 'r'))
```

Comment:
This is no longer correct, because the sampled pairs file is now a `.jsonl` file.